### PR TITLE
doc: add governance for adding regular members

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by mhdawson** [comment]
Refs https://github.com/openjs-foundation/cross-project-council/issues/263

**Comment by mhdawson** [comment]
@brianwarner I looked for any other mailing lists on https://lists.openjsf.org/g/main/subgroups.  Is there any mailing lists that they need to be added to?

**Comment by brianwarner** [comment]
> @brianwarner I looked for any other mailing lists on https://lists.openjsf.org/g/main/subgroups. Is there any mailing lists that they need to be added to?

@mhdawson We currently have the cpc-private list for highly sensitive topics, but that currently only has voting members.  I can certainly add the regular members, but would appreciate some guidance from the CPC before I do so.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -103,6 +103,26 @@ These members will be elected for a term of 1 year as follows:
...
+
+The PR to add a regular member is approved when:
+
+* there are no outstanding objections
+* there are two or more approvals by voting CPC members
+* the PR has been opened for at least 1 week
+
+Once a PR is ready to be landed, the CPC member who lands the pull request should:
+
+* Send a notification to the project contacts for the project indentified in the PR
```
Comment (MylesBorins) [inline GOVERNANCE.md]: Do individuals need to be representing a project as part of their CPC membership?

File: GOVERNANCE.md [hunk 2]
```diff
@@ -103,6 +103,26 @@ These members will be elected for a term of 1 year as follows:
...
+  indicating that a new regular CPC member has joined the CPC on behalf of the project.
+* Add the member to the github `cpc` [team](https://github.com/orgs/openjs-foundation/teams/cpc)
```
Comment (MylesBorins) [inline GOVERNANCE.md]: there is a sub team for regular members
```suggestion
* Add the member to the github `cpc-regular-members` [team](https://github.com/orgs/openjs-foundation/teams/cpc-regular-members)
```

File: GOVERNANCE.md (hunk 1 above)
Comment (mhdawson) [reply GOVERNANCE.md]: The current governance says they must be part of one of the projects.  
>  Anyone who has been a member of one of the projects under the OpenJS Foundation for at least three months may request to become a regular member by opening a PR to add themselves to the list of regular members. 

https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#regular-members

None of the text says they are "representing" the project although that is possibly implied.  The notification what we agreed to in terms of validating that they have been active in one of the projects for 3 months.

Comment (eemeli) [inline GOVERNANCE.md]:
```suggestion
* Send a notification to the project contacts for the project identified in the PR
```

**Pull request merged**